src/cgame/cg_buildable.cpp

```diff
@@ -630,6 +630,7 @@ void CG_InitBuildables()
 		//Prefer md5 models over md3
 
 		if ( cg_highPolyBuildableModels.integer &&
+		     CG_FileExists( va( "models/buildables/%s/%s.iqm", buildableName, buildableName ) ) &&
```

Comment:
Maybe now it's a good idea to store the result of va(...) in a variable, we don't want to risk divergence between the condition and the actual value.